src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -107,6 +121,10 @@ export default class AddNameTranslation extends Vue {
     })
   }
 
+  private get isAddingTranslation () {
+    return this.editNameTranslation === ''
+  }
+
```

Comment:
Although not required, minor doc-block comments are nice, even for the most simple getters.
In this case, it is a little curious how an empty string is defining if an edit is in progress. 
So something like `/** Returns true when there is no name to edit. */` or however you would best describe it :) 

Your getter name already gives away the return type ⭐  but doesn't hurt to define it :)

Reply:
Added a comment here

Reply:
Thanks! 
Super small thing, but the return type as well 😁 
ie `isAddingTranslation (): boolean {}`